src/components/RichEditor/RichEditor.test.jsx

```diff
@@ -58,43 +61,53 @@ describe('RichEditor', () => {
     expect(container).toMatchSnapshot();
   });
 
-  it.skip('change handlers are called', () => {
-    // TODO: Update the test to work with RTL
+  it('change handlers are called', async () => {
     const onChange = jest.fn();
-    const mockEditor = {
-      getContent: jest.fn().mockImplementation(() => 'Hello, text'),
-    };
+    const mockHandleEditorChange = jest.spyOn(RichEditor.prototype, 'handleEditorChange');
 
-    const component = render(<RichEditor
+    const { container } = render(<RichEditor
       label="Rich Text Editor Test"
       id="rich-text-editor-test"
       input={{
-        value: null,
+        value: 'Start',
         onChange,
       }}
       maxChars={500}
       meta={{ touched: true, error: 'Required' }}
     />);
-    const instance = component.instance();
 
-    expect(instance.state.charCount).toEqual(0);
-    expect(instance.state.value).toBeNull();
+    // Initial render
+    await waitFor(() => expect(container.querySelector('iframe').contentDocument.querySelector('#tinymce')).toBeInTheDocument());
+    expect(await screen.findByText('Recommended character limit (including spaces) is 500. 495 characters remaining.')).toBeInTheDocument();
+    await waitFor(() => expect(container.querySelector('iframe').contentDocument.querySelector('#tinymce').querySelector('p').textContent).toBe('Start'));
+
+    // JSDOM does not support these, so stub them out
```

Comment:
https://github.com/jsdom/jsdom/issues/3729

Reply:
The error is raised somewhere inside tinymce.

<img width="990" alt="Screenshot 2025-05-27 at 1 29 49 AM" src="https://github.com/user-attachments/assets/4a014bc0-1b73-4061-ae8b-b4aa46077b1c" />